server/leader.go

```diff
@@ -36,6 +35,8 @@ import (
 // The timeout to wait transfer etcd leader to complete.
 const moveLeaderTimeout = 5 * time.Second
 
+const leaderTickInterval = 50 * time.Millisecond
```

Comment:
Maybe add some comments about the relation of `leaderTickInterval` and `tso.UpdateTimestampStep`?

Reply:
Actually no special relationship, they can run independently.

Reply:
`leaderTickInterval` controls the frequency of calling `UpdateTimestamp`. And in `tso` package, it assumes the frequency is controlled by `tso.UpdateTimestampStep`.

Reply:
Updated. Use different ticker now 😄